src/lib/logger.ts

```diff
@@ -0,0 +1,62 @@
+import { format, transports, createLogger, Logger } from 'winston';
+import DailyRotateFile from "winston-daily-rotate-file";
```

Comment:
change this to what we spoke about to allow the app to work in the absence of a logging config